src/release/breaking-changes/index.md

```diff
@@ -30,11 +30,13 @@ release, and listed in alphabetical order:
 
 ### Not yet released to stable
 
+* [Android Java Gradle migration guide][]
```

Comment:
hmm this feels weird to be under not yet release to stable, currently there is nothing in the guide that depends on the next stable release for developers to follow it and unblock themselves when using Flamingo.

if we add the flutter analyze --suggestions tool steps into the guide then  it has something that is not yet released to stable

Reply:
Yeah. I can just take it out of the index.